code/modules/jobs/job_types/captain.dm

```diff
@@ -50,6 +50,7 @@ Captain
 	duffelbag = /obj/item/storage/backpack/duffelbag/captain
 
 	implants = list(/obj/item/implant/mindshield)
+	mind_traits = list(TRAIT_DISK_VERIFIER)

```

Comment:
Feels iffy to me that this is on the outfit, maybe it should go on the job instead?